### PR TITLE
Compute Engine with Cloud SQL sample

### PR DESCRIPTION
**Pull request by dmytro-grankin**
This PR brings a new sample, which shows how to deploy a server to Google Compute Engine. The server uses a database within Google Cloud SQL.

Additionally, `LocalClient` was renamed to `ClientApp`. From now, the Gradle task for running the client allows to configure a hostname and port to establish a connection with the `gRPC` server.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@armiol PTAL.

File: deployment/compute-cloud-sql/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# https://github.com/GoogleCloudPlatform/openjdk-runtime
```
[the remark this answers is not in the document]
Comment (dmytro-grankin) [reply deployment/compute-cloud-sql/Dockerfile]: `MAINTAINER` is [deprecated](https://github.com/lukasmartinelli/hadolint/wiki/DL4000).

File: deployment/compute-cloud-sql/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# compute-cloud-sql
+
+A server for deploying to Google Compute Engine, which uses a remote database within Google Cloud SQL.
+
+### Configure a Cloud Platform project
+
+1. [Select or create](https://console.cloud.google.com/cloud-resource-manager)
```
Comment (armiol) [inline deployment/compute-cloud-sql/README.md]: Why in this file there are only `1.`-numbered items?

File: deployment/compute-cloud-sql/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (armiol) [reply deployment/compute-cloud-sql/Dockerfile]: @dmytro-grankin please report this to Codacy support. Also, please remove this inspection from our profile for now.

**Pull request merged**